Assets/MixedRealityToolkit/_Core/Services/MixedRealityToolkit.cs

```diff
@@ -550,21 +560,28 @@ public bool RegisterService(Type type, IMixedRealityService service)
         /// <summary>
         /// Generic function used to retrieve a service from the Mixed Reality Toolkit active service registry
         /// </summary>
+        /// <param name="showLogs">Should the logs show when services cannot be found?</param>
         /// <typeparam name="T">The interface type for the system to be retrieved.  E.G. InputSystem, BoundarySystem.
         /// *Note type should be the Interface of the system to be retrieved and not the class itself</typeparam>
         /// <returns>The instance of the service class that is registered with the selected Interface</returns>
-        public T GetService<T>() where T : IMixedRealityService
+        public T GetService<T>(bool showLogs = true) where T : IMixedRealityService
         {
-            return (T)GetService(typeof(T));
+            return (T)GetService(typeof(T), showLogs);
         }
 
         /// <summary>
         /// Retrieve a service from the Mixed Reality Toolkit active service registry
         /// </summary>
         /// <param name="type">The interface type for the system to be retrieved.  E.G. InputSystem, BoundarySystem</param>
+        /// <param name="showLogs">Should the logs show when services cannot be found?</param>
         /// <returns>The Mixed Reality Toolkit of the specified type</returns>
-        public IMixedRealityService GetService(Type type)
+        public IMixedRealityService GetService(Type type, bool showLogs = true)
         {
+            if (isApplicationQuitting)
+            {
+                return null;
+            }
+
             if (ActiveProfile == null)
```

Comment:
need to check showLogs here and other similar calls

Reply:
Why? These are much more important than the other where we check. This sort of thing is pretty serious and we should def always show this error no matter what.

Reply:
Is that not the purpose of the showLogs argument? If a customer passes in "showLogs == false" it seems as if the method should not log.

This is where something like the logging level task/proposal can address the "much more important". It gives customers a finer grain control over the types of messages received.